cpp/src/rolling/rolling_detail.hpp

```diff
@@ -30,9 +30,11 @@ namespace detail {
 template <typename ColumnType, class AggOp, aggregation::Kind op>
 static constexpr bool is_rolling_supported()
 {
-  if (!cudf::detail::is_valid_aggregation<ColumnType, op>()) { return false; }
+  if (!cudf::detail::is_valid_aggregation<ColumnType, op>()) {
+    return false;
+  } else
 
-  if (cudf::is_numeric<ColumnType>()) {
+    if (cudf::is_numeric<ColumnType>()) {
     constexpr bool is_comparable_countable_op = std::is_same<AggOp, DeviceMin>::value or
```

Comment:
Does this pass clang-format checks? Seems like line  38 needs  to be indented  further now that 37 is.

Reply:
So `clang-format` did not break the `else if` - that had to be done by the coder. And I guess this isn't something `clang-format` handles. Not sure it is necessarily a bug, but definitely does seem like something `clang-format` should account for. I checked `clang-format` 10 on https://zed0.co.uk/clang-format-configurator/ and it has the same behaviour.

Reply:
@karthikeyann I think it would be nice to *not* break  the `else if` across 3 lines but I won't  let that block this PR if you have a reason to do it.

Reply:
My guess is this was a typo and not intentional

Reply:
This is not intentional. I just added else and used `clang-format`. I thought, `clang-format` would take care of it. I will fix it and merge.